src/tcgwars/logic/impl/gen3/PowerKeepers.groovy

```diff
@@ -619,10 +619,7 @@ public enum PowerKeepers implements LogicCardInfo {
           delayedA{
             before KNOCKOUT, {
               if (bg.em().retrieveObject("Energy_Grounding") != bg.turnCount && (ef as Knockout).byDamageFromAttack && bg.currentTurn==self.owner.opposite && ef.pokemonToBeKnockedOut != self && ef.pokemonToBeKnockedOut.cards.filterByType(BASIC_ENERGY).energyCount(C)) {
-                if (confirm("Move an energy from ${ef.pokemonToBeKnockedOut} to $self?", self.owner)) {
-                  bg.em().storeObject("Energy_Grounding", bg.turnCount)
```

Comment:
Don't you still need this to avoid attaching energy from multiple knockouts in the same turn?

Reply:
I think storing the turn count will only allow one activation per turn or are you talking about something else?

Reply:
Where is it storing the turn count now that this line has been removed?

Reply:
TY! I'm actually blind.